.gitignore

```
@@ -0,0 +1,3 @@
+easy_manipulation_deployment/grasp_planner/.vscode/*
+easy_manipulation_deployment/workcell_builder/workcell_builder/include/attributes/.vscode/*
```

Combine vscode ignore.
```suggestion
.vscode/
```